### ISSUE TITLE
Ask contribution to Tensorflow addons for general scope utils, loss, layers, ops

### ISSUE THREAD

**Issue by bhack** [tensorflow/community#223]
As we have just refreshed the model repo as model garden I would enforce the contributions policies of generale use (or already established in literature utils, losses, layers, ops to be contribute more systematically in Tensorflow/addons instead to be embedded or duplicated in the model repos.
E.g:
https://github.com/tensorflow/addons/issues/361#issuecomment-606856705
https://github.com/tensorflow/addons/issues/1364#issuecomment-602287778
https://github.com/tensorflow/addons/issues/1366
https://github.com/tensorflow/addons/issues/1226#issuecomment-599483920
https://github.com/tensorflow/addons/issues/1226#issuecomment-600204811

 /cc @ewilderj  @facaiy @seanpmorgan 

# Policies to be enforced with a PR
https://github.com/tensorflow/models/blob/master/CONTRIBUTING.md
https://github.com/google-research/google-research/blob/master/CONTRIBUTING.md
https://github.com/google/automl/blob/master/CONTRIBUTING.md

**Comment by bhack** [comment]
See also the approved RFC https://github.com/tensorflow/community/blame/master/rfcs/20190802-model-garden-redesign.md#L60-L61

**Comment by bhack** [comment]
I will add in the target contributions repos the new keras-cv and keras-nlp.

**Comment by bhack** [comment]
E.g. See https://github.com/tensorflow/models/pull/8962#pullrequestreview-457058720

**Comment by theadactyl** [comment]
@bhack still interested in this?

**Comment by bhack** [comment]
In general this is a still valid topic but:

- For TF Addons we could close this as we have:
https://github.com/keras-team/keras-cv/issues/74
https://github.com/keras-team/keras-nlp/issues/11

- More in general for the ecosystem I could add a reference to this ticket in  https://github.com/tensorflow/community/pull/384
